tests/utils.go

```diff
@@ -548,13 +548,13 @@ func NewRandomVirtualMachineInstanceWithBlockDisk(imageUrl, namespace string, ac
 	return NewRandomVirtualMachineInstanceWithDisk(imageUrl, namespace, sc, accessMode, k8sv1.PersistentVolumeBlock)
 }
 
-func NewRandomVMI() *v1.VirtualMachineInstance {
-	// To avoid mac address issue in the tests change the pod interface binding to masquerade
-	// https://github.com/kubevirt/kubevirt/issues/1494
-	vmi := libvmi.New(
+func NewRandomVMI(opts ...libvmi.Option) *v1.VirtualMachineInstance {
```

Comment:
I do not understand why we would like to extend this function. We are interested to drop it in favor of `libvmi`.
Or replace it at the callers when you need these options.